src/settings.ts

```diff
@@ -102,6 +102,7 @@ export interface ISettings {
     cwd?: string;
     notebooks?: INotebooksSettings;
     enableReferencesCodeLens?: boolean;
+    analyzeOpenDocumentsOnly?: boolean;
```

Comment:
btw I picked this name specifically to match C#'s

It doesn't perfectly fit, but I think it's pretty accurate. Open to changing though.

Reply:
LGTM.